src/fundus/scraping/crawler.py

```diff
@@ -464,14 +465,22 @@ def load_paths(url: str) -> List[str]:
                     bar.update()
                     return paths
 
+            def random_sleep(func: Callable[_P, _T], between: Tuple[float, float]) -> Callable[_P, _T]:
+                @wraps(func)
+                def wrapper(*args: _P.args, **kwargs: _P.kwargs) -> _T:
+                    time.sleep(random.uniform(*between))
+                    return func(*args, **kwargs)
+
+                return wrapper
+
             if self.processes == 0:
                 nested_warc_paths = [load_paths(url) for url in urls]
             else:
                 # use two threads per process, default two threads per core
                 max_number_of_threads = self.processes * 2
 
                 with ThreadPool(processes=min(len(urls), max_number_of_threads)) as pool:
-                    nested_warc_paths = pool.map(load_paths, urls)
+                    nested_warc_paths = pool.map(random_sleep(load_paths, (0, 3)), urls)
```

Comment:
I was thinking it might make sense to have a minimum wait of more than 0. Or was your approach that in case it crashes due to the SlowDown from CC that it'll be retried anyway after a short break, in which case it should go through again

Reply:
My Idea was to spread the requests so we wouldn't bombard the server with min(process*2, 94) concurrent requests. So far it seems to work. Unfortunately, I don't understand the second part of your comment. We do not retry any connection here.

Reply:
Nevermind that, I think I had the PR in mind this was based on. Ok, it does seem to work. I was surprised though, because when I had a 5 second manual delay it still crashed yesterday, but I couldn't reproduce it earlier. Maybe it was my implementation